whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestBasicAuthTestIntegration.java

```diff
@@ -0,0 +1,273 @@
+            "nic-hdl:        TP1-TEST\n" +
+            "mnt-by:         OWNER-MNT\n" +
+            "source:         TEST\n";
+
+    public static final RpslObject TEST_PERSON = RpslObject.parse(TEST_PERSON_STRING);
+    private static final RpslObject PAULETH_PALTHEN = RpslObject.parse("" +
+            "person:    Pauleth Palthen\n" +
+            "address:   Singel 258\n" +
+            "phone:     +31-1234567890\n" +
+            "e-mail:    noreply@ripe.net\n" +
+            "mnt-by:    OWNER-MNT\n" +
+            "nic-hdl:   PP1-TEST\n" +
+            "remarks:   remark\n" +
+            "source:    TEST\n");
+
+    private static final RpslObject OWNER_MNT = RpslObject.parse("" +
+            "mntner:      OWNER-MNT\n" +
+            "descr:       Owner Maintainer\n" +
+            "admin-c:     TP1-TEST\n" +
+            "upd-to:      noreply@ripe.net\n" +
+            "auth:        MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
+            "auth:        SSO person@net.net\n" +
+            "mnt-by:      OWNER-MNT\n" +
+            "source:      TEST");
+
+    private static String TEST_ROLE_STRING = "" +
+            "role:           Test Role\n" +
+            "address:        Singel 258\n" +
+            "phone:          +31 6 12345678\n" +
+            "nic-hdl:        TR1-TEST\n" +
+            "admin-c:        TR1-TEST\n" +
+            "abuse-mailbox:  abuse@test.net\n" +
+            "mnt-by:         OWNER-MNT\n" +
+            "source:         TEST\n";
+    private static final RpslObject TEST_ROLE = RpslObject.parse(TEST_ROLE_STRING);
+
+    private static final RpslObject TEST_IRT = RpslObject.parse("" +
+            "irt:          irt-test\n" +
+            "address:      RIPE NCC\n" +
+            "e-mail:       noreply@ripe.net\n" +
+            "admin-c:      TP1-TEST\n" +
+            "tech-c:       TP1-TEST\n" +
+            "auth:         MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
+            "mnt-by:       OWNER-MNT\n" +
+            "source:       TEST\n");
+
+    @Autowired private WhoisObjectMapper whoisObjectMapper;
+    @Autowired private TestDateTimeProvider testDateTimeProvider;
+    @BeforeEach
+    public void setup() {
+        databaseHelper.addObject("person: Test Person\nnic-hdl: TP1-TEST");
+        databaseHelper.addObject("role: Test Role\nnic-hdl: TR1-TEST");
+        databaseHelper.addObject(OWNER_MNT);
+        databaseHelper.updateObject(TEST_PERSON);
+        databaseHelper.updateObject(TEST_ROLE);
+        testDateTimeProvider.setTime(LocalDateTime.parse("2001-02-04T17:00:00"));
+    }
+
+    @Test
+    public void lookup_mntner_incorrect_basic_auth_password_without_unfiltered_param_is_fully_filtered() {
+        final WhoisResources whoisResources = SecureRestTest.target(getSecurePort(), "whois/test/mntner/OWNER-MNT")
+                .request()
+                .header("Authorization", getBasicAuthenticationHeader("test", "incorrect"))
```

Comment:
HttpHeaders.AUTHORIZATION ? instead of Authorization?

Reply:
I do not like putting two lines when it can be done is single line , also its much clearer with .header then property

Reply:
Ok, it is more readable on your way I think yes
but could you change "Authorization" by HttpHeaders.AUTHORIZATION?